packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -480,6 +480,31 @@ const api = new apigateway.RestApi(this, 'books', {
   }
 })
 ```
+### Access Logging
+
+The following example will configure API Gateway to enable custom access logging
+
+```ts
+const api = new apigateway.RestApi(this, 'books', {
+  deployOptions: {
+    accessLogSetting: {
+        destinationArn: 'arn:aws:logs:us-east-1:123456789012:log-group:sample_group_name',
```

Comment:
I think this should be:

 `destinationArn: new apigateway.CloudWatchLogsDestination('arn:aws:logs:us-east-1:123456789012:log-group:sample_group_name');`

Reply:
Actually, on fiddling in my IDE, due to the interface input into `apigateway.CloudWatchLogsDestination`, inputting an arn is difficult.  We should fully expand the example to this:

```ts
    const loggroup = new cwlogs.LogGroup(this, "ApiGatewayAccessLogs", {});
    const restApi = new apigw.LambdaRestApi(this, 'books', {
      deployOptions: {
        accessLogDestination: new CloudWatchLogsDestination(loggroup),
        accessLogFormat: JSON.stringify({
          requestId: "$context.requestId",
          ip: "$context.identity.sourceIp",
          caller: "$context.identity.caller",
          user: "$context.identity.user",
          requestTime: "$context.requestTime",
          httpMethod: "$context.httpMethod",
          resourcePath: "$context.resourcePath",
          status: "$context.status",
          protocol: "$context.protocol",
          responseLength: "$context.responseLength"
        })
      }
    });
```

Let me know your thoughts :)

Reply:
You're right.
I forgot to revise the document.
The document has been modified, thank you!